utils/obsproc/superob.h

```diff
@@ -8,11 +8,47 @@
 
 namespace gdasapp {
   namespace superobutils {
+    // Function to compute distance between two points
+    struct GeoPoint {
```

Comment:
make use of `eckit::geometry::Point2` instead of declaring a new data structure.